cibuildwheel/docker_container.py

```diff
@@ -117,11 +117,12 @@ def glob(self, path: PurePath, pattern: str) -> List[PurePath]:
 
         return [PurePath(p) for p in path_strs]
 
-    def call(self, args: Sequence[Union[str, PathLike]], env: Dict[str, str] = {},
+    def call(self, args: Sequence[Union[str, PathLike]], env: Optional[Dict[str, str]] = None,
```

Comment:
This is the one manual change - using a mutable default argument was caught by bugbear.